### PR TITLE
Hide test logs in Teamcity

### PR DESCRIPTION
**Pull request by wolfs**

### CODE REVIEW

**Comment by wolfs** [comment]
Seems to work: https://builds.gradle.org/buildConfiguration/Gradle_Check_Stage_QuickFeedbackLinuxOnly_Trigger/36823721?buildTab=artifacts

**Comment by wolfs** [comment]
We should probably put this change on `release` and `master`.

File: .teamcity/common/extensions.kt [hunk 1]
```diff
@@ -60,11 +60,11 @@ fun VcsSettings.filterDefaultBranch() {
 
 fun BuildType.applyDefaultSettings(os: Os = Os.linux, timeout: Int = 30, vcsRoot: String = "Gradle_Branches_GradlePersonalBranches") {
     artifactRules = """
-        build/report-* => .
-        buildSrc/build/report-* => .
-        subprojects/*/build/tmp/test files/** => test-files
-        subprojects/*/build/tmp/test files/** => test-files
-        build/errorLogs/** => errorLogs
+        build/report-* => .teamcity/
```
Comment (blindpirate) [inline .teamcity/common/extensions.kt:63]: In this way, how could we get the test files when we want to debug issues?

Comment (blindpirate) [reply .teamcity/common/extensions.kt:63]: ok, I see the "Hidden artifact" section in TeamCity documentation.

Comment (wolfs) [reply .teamcity/common/extensions.kt:63]: Exactly. IIUC, guest users can't see the hidden artifacts.See https://teamcity-support.jetbrains.com/hc/en-us/community/posts/206890335-Downloading-hidden-artifacts-as-guest-user.

I didn't find any more documentation on the access rights to hidden artifacts.

What I just found is that we need to adjust the cleanup patterns to include hidden artifacts, if not they won't be cleaned up.

**Comment by wolfs** [comment]
Example for the new location: https://builds.gradle.org/buildConfiguration/Gradle_Check_Quick_1_bucket41/36843873?buildTab=artifacts

**Comment by wolfs** [comment]
@blindpirate Please take another look. I adjusted the location and added a cleanup rule, since hidden artifacts are not cleaned up by default.

**Pull request merged**